python/foglamp/services/core/api/browser.py

```diff
@@ -212,6 +213,43 @@ def prepare_limit_skip_payload(request, _dict):
     return web.json_response(response)
 
 
+async def asset_all_readings_summary(request):
+    """ Browse all the assets for which we have recorded readings and
+    return a summary for all sensors values for an asset code. The values that are
+    returned are the min, max and average values of the sensor.
+
+    :Example:
+            curl -sX GET http://localhost:8081/foglamp/asset/fogbench_humidity/summary
+    """
+    try:
+        # Get readings from asset_code
+        asset_code = request.match_info.get('asset_code', '')
+        payload = PayloadBuilder().SELECT("reading").WHERE(["asset_code", "=", asset_code]).payload()
+        _readings = connect.get_readings_async()
+        results = await _readings.query(payload)
+        # Find keys in readings
+        reading_keys = list(results['rows'][0]['reading'].keys())
+        response = []
+        for reading in reading_keys:
```

Comment:
could you please file a ticket for storage support need, so that we can avoid this loop, it is all from `readings` table only.

Reply:
https://scaledb.atlassian.net/browse/FOGL-1768